catalyst/contrib/transforms/tensor.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+from albumentations.core.transforms_interface import ImageOnlyTransform
+import torch
+
+
+class TensorToImage(ImageOnlyTransform):
```

Comment:
why do we need this transforms here?
@bagxi what do you think?

Reply:
btw, we also need to merge this PR first
https://github.com/catalyst-team/catalyst/pull/604

Reply:
This transformation is needed when torch.Tensor is returned from the dataset, not numpy. For example, MNIST in our tests

Reply:
maybe, it should be `TensorToNumpy` then?

Reply:
or just `ToNumpy`?

Reply:
it's okay to me, but this transform was already in catalyst.

so we need @bagxi  to discuss

Reply:
and what about `TensorToImage` naming?

Reply:
and would it be useful if we will add parameter like `use catalyst.utils.tensor_to_ndimage`?

Reply:
let's make `tensor_to_ndimage` more general with flags like `need_normalization` and `need_move_channels_dim` and then use this function inside of `TensorToImage`/`ToNumpy`/`YouNameIt` transfrom

Reply:
prosed simplification, `normalize=`, `move_channels_dim=`

Reply:
maybe it is better to use `denormalize=` (instead of `normalize=`) as in `utils.tensor_to_ndimage` we are multiplying image by std and adding mean?

Reply:
sure